sentry/src/main/java/io/sentry/util/MapObjectWriter.java

```diff
@@ -151,6 +156,12 @@ public MapObjectWriter value(final @Nullable String value) throws IOException {
     return this;
   }
 
+  @Override
+  public ObjectWriter jsonValue(@Nullable String value) throws IOException {
```

Comment:
What's the reason for adding this?

Reply:
sorry this sneaked in from the target branch, I've updated it now to remove from this PR